src/pages/Auth.tsx

```diff
@@ -1,5 +1,25 @@
+import { Form } from "react-router-dom";
+import { COMMON_ROUTES_NAMES } from "@/routing";
+
+import useCurrentUser from "@/hooks/useCurrentUser";
+
 const Auth = () => {
-	return <h1>Auth</h1>
-}
+	const actionData = useCurrentUser();
```

Comment:
how come that the hook that by the name should return somehting user-related, returns `actionData`

Reply:
it is good that the actions from the react-router are used here, but the naming...